models/schema.yml

```diff
@@ -279,3 +281,199 @@ models:
         description: "The number of units in inventory of the product variant"
         tests:
           - not_null
+
+  - name: shopify__refunds
+    description: "A table storing information regarding order refunds"
+    columns:
+      - name: refund_id
+        description: "The unique identifier of the refund"
+        tests:
+          - unique
+          - not_null
+      - name: order_id
+        description: "The unique identifier of the order"
+        tests:
+          - not_null
+          - relationships:
+              to: ref('shopify__orders')
+              field: order_id
+      - name: user_id
+        description: "The unique identifier of the user who created the refund"
+        tests:
+          - not_null
+      - name: total_price
+        description: "The total price of the original order"
+        tests:
+          - not_null
+
+metrics:
+  - name: number_of_customers
+    label: Number of Customers
+    model: ref('shopify__customers')
+    description: "The number of customers"
+
+    type: count_distinct
+    sql: customer_id
+
+    timestamp: created_at_timestamp
+    time_grains: [ day, week, month, year ]
+
+    dimensions:
+      - city
+      - country
+
+  - name: number_of_repeat_customers
+    label: Number of Repeat Customers
+    model: ref('shopify__customers')
+    description: "The number of repeat customers"
+
+    type: count_distinct
+    sql: customer_id
+
+    timestamp: created_at_timestamp
+    time_grains: [ day, week, month, year ]
+
+    dimensions:
+      - city
+      - country
+      - email
+      - first_name
+      - last_name
+      - orders_count
+      - total_spent
+
+  - name: number_of_orders
+    label: Number of Orders
+    model: ref('shopify__orders')
+    description: "The number of orders"
+
+    type: count_distinct
+    sql: order_id
+
+    timestamp: created_at_timestamp
+    time_grains: [ day, week, month, year ]
+
+    dimensions:
+      - currency
+      - customer_id
```

Comment:
not sure I would want to split by customer_id, discounts or price since it will basically give me single results

Reply:
I didn't add those so we could split by them but rather so they are available in the tables. So if you wanted to be able to see which customers have the lowest/highest average order value

<img width="1715" alt="Screenshot 2022-05-09 at 09 28 28" src="https://user-images.githubusercontent.com/5059404/167370781-50166b6f-dd39-4f14-8211-6ba130e5658d.png">